### DISCUSSION TITLE
Question on reproducing the results

### DISCUSSION THREAD

**Discussion by anyangml** [janosh/matbench-discovery#142] [Q&A] [answered]
First and foremost, I would like to extend my sincere appreciation for your dedication to assembling such an impressive collection of models within your repository, as well as for your continued efforts to maintain their currency. Your work is invaluable to the community.

Upon utilizing the resources provided, I have encountered a concern I hope you can help me address. Despite having cloned the latest version of the main branch and ensuring all dependencies are correctly installed, I've observed a slight discrepancy between the outcomes produced by the `preds.py` script (I printed the `df_metrics`) and the results published on your website.

Would you be able to direct me to any additional steps or considerations I might have overlooked which could account for this difference? Any guidance you could provide would be greatly appreciated.

Thank you for your time and assistance.

Best regards,
Anyang
<img width="1193" alt="image" src="https://github.com/user-attachments/assets/27d9261c-c464-4288-ad95-a3ddef13e7ba">

<img width="1156" alt="image" src="https://github.com/user-attachments/assets/0784e56d-fa25-4b97-93ec-0c2f56c8a482">

**Comment by janosh** [comment, marked as answer]
by default, the site shows the metrics on the subset of the WBM test set that includes only unique structure prototypes (removing prototype overlap with MP as well as duplicates within WBM itself). while this only has a minor effect on the metrics, our goal with that was to maximize the OODness of the test set. if you print `df_metrics_uniq_protos` instead, that should match what you see on the site. see https://github.com/janosh/matbench-discovery/pull/75, https://github.com/janosh/matbench-discovery/discussions/69#discussioncomment-10545599, https://github.com/janosh/matbench-discovery/discussions/69#discussioncomment-10546116 for additional context
